malfeasance/handler_test.go

```diff
@@ -187,9 +187,10 @@ func TestHandler_HandleMalfeasanceProof(t *testing.T) {
 		err := h.HandleMalfeasanceProof(context.Background(), "peer", codec.MustEncode(gossip))
 		require.ErrorIs(t, ErrKnownProof, err)
 
-		malProof, err := identities.GetMalfeasanceProof(h.db, nodeID)
-		require.NoError(t, err)
-		malProof.SetReceived(time.Time{})
+		var blob sql.Blob
+		require.NoError(t, identities.LoadMalfeasanceBlob(context.Background(), h.db, nodeID.Bytes(), &blob))
+		malProof := &wire.MalfeasanceProof{}
+		codec.MustDecode(blob.Bytes, malProof)
```

Comment:
This is repeated so often, maybe it deserves a helper in `malfeasance` package?

```go
func LoadMalfeasanceProof(db sql.Executor, id types.NodeID) (*mwire.MalfeasanceProof, error) { ... }
```

Reply:
I thought about that as well. The reason I didn't implemented a function that returns a `wire.MalfeasanceProof` from the DB directly is because on the one hand this function would only used in tests and arguably it the tests instead should assert that an identity has been marked as malfeasant instead of fetching the proof from the DB and asserting that a specific proof was stored (separation of concerns).

Wdyt about simplifying the relevant tests to only check for malfeasance instead of asserting that a specific proof was stored?